### DISCUSSION TITLE
502 when authenticating using google

### DISCUSSION THREAD

**Discussion by bagheriali2001** [outline/outline#4362] [Self Hosting] [answered]
Hi, guys.
I am trying to implement outline for our company using docker version. But in both my own testing lab and in company deployment I get a 502 Nginx error only when trying to authenticate using google ( we can not use another form of authentication ). At first, I thought maybe the `https-portal` service from this [docker compose](https://app.getoutline.com/s/770a97da-13e5-401e-9f8a-37949c19f97e/doc/docker-7pfeLP5a8t) doesn't work properly but when I tested it on company server (with [custom Nginx config](https://app.getoutline.com/s/770a97da-13e5-401e-9f8a-37949c19f97e/doc/nginx-6htaRboR57) and commented two `proxy_set_header` line due do [this error and answer](https://github.com/outline/outline/issues/1985#issuecomment-813507285) ) and it's still the same.
Any body has any idea how to fix this?

UPDATE:
I found the problem behind it. It was due to restrictions of google APIs.

**Comment by bagheriali2001** [comment, marked as answer]
I found the problem behind it. It was due to restrictions of google APIs.

**Reply by mweth** [reply]
What restrictions?
